pkg/cmd/main/main.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 // test rest endpoint
+
```

Comment:
I am guessing this file is not used anymore?  It looks like it's listening on port 9443, but I thought you were listening on 8080 now.

Reply:
true, this was just when we got started, I'll delete it